src/fs.ts

```diff
@@ -71,7 +71,7 @@ export class LocalFileSystem implements FileSystem {
 		return path.resolve(this.rootPath, uri2path(uri));
 	}
 
-	async getWorkspaceFiles(base?: string): Promise<string[]> {
+	async getWorkspaceFiles(base: string): Promise<string[]> {
```

Comment:
This must stay optional, I fixed the bug in my other branch:
https://github.com/sourcegraph/javascript-typescript-langserver/pull/108/files#diff-a987912cd3148bf666f759bdbf896718R66